packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -13,11 +13,14 @@ flutter:
         pluginClass: UrlLauncherPlugin
       ios:
         pluginClass: FLTUrlLauncherPlugin
+      web:
+        default_package: url_launcher_web
 
 dependencies:
   flutter:
     sdk: flutter
   url_launcher_platform_interface: ^1.0.1
+  url_launcher_web: any
```

Comment:
I'd be specific on this one, but we'll see how 'any' goes for a while!

Reply:
See discussion on https://flutter.dev/go/federated-plugins we want this edge to be any